sysutils/jello/Portfile

```diff
@@ -0,0 +1,37 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem              1.0
+PortGroup               python 1.0
+
+name                    jello
+version                 1.5.4
+revision                0
+
+homepage                https://kellyjonbrazil.github.io/jello/
+
+description             Filter JSON and JSON Lines data with Python syntax
+
+long_description        Jello is similar to jq in that it processes JSON and \
+                        JSON Lines data, except Jello uses standard Python \
+                        dict and list syntax.
+
+categories              sysutils textproc python
+license                 MIT
+maintainers             {ajhall.us:macports @ajhall} \
+                        openmaintainer
+supported_archs         noarch
+
+checksums               rmd160  265e60aabc0d550e4cd30405ba0d145277b42aef \
+                        sha256  6e536485ffd7a30e4d187ca1e2719452e833f1939c3b34330d75a831dabfcda9 \
+                        size    25574
+
+python.default_version  310
+
+depends_lib-append      port:py${python.version}-pygments
+
+post-destroot {
+    # Install man page
+    xinstall -d ${destroot}${prefix}/share/man/man1/
```

Comment:
I *think* MacPorts already makes this directory for you, so you could test and remove this line and see if it still works

Reply:
Ah, yep! Works fine without that line.